src/core/utils/Utilities.ts

```diff
@@ -170,3 +170,22 @@ export const decodeBlock = (input: any, inputOffset: number, output: any, output
     output[outputOffset + 3] = ((bytes[4] & 0x01) << 7) | (bytes[5] << 2) | (bytes[6] >> 3);
     output[outputOffset + 4] = ((bytes[6] & 0x07) << 5) | bytes[7];
 };
+
+export const arrayDeepEqual = (fist, second, numElementsToCompare?): boolean => {
```

Comment:
fist => first.

if you want to add tests for this, you can grab them from: https://github.com/symbol/catapult-rest/blob/dev/catapult-sdk/test/utils/arrayUtils_spec.js#L138

Reply:
cheers @Jaguar0625  will add them 👍